app/task/project/saveotio/saveotio.cpp

```diff
@@ -135,12 +135,29 @@ OTIO::Track *SaveOTIOTask::SerializeTrack(Track *track)
     if (dynamic_cast<ClipBlock*>(block)) {
       auto otio_clip = new OTIO::Clip(block->GetLabel().toStdString());
 
+      double rate = static_cast<ClipBlock*>(block)->connected_viewer()->GetVideoParams().frame_rate().toDouble();
+      if (rate == qSNaN()) {
```

Comment:
Using the macro `qIsNaN()` is preferable to an `==` comparison

Reply:
Thanks, done